Dockerfile

```
@@ -1,5 +1,5 @@
 # use KDUX base Rails image, configure only project-specific items here
-FROM broadinstitute/kdux-rails-baseimage:1.9
+FROM singlecellportal/rails-baseimage:0.2.3
```

version 1.0.0 is now available
```suggestion
FROM singlecellportal/rails-baseimage:1.0.0
```